docs/user_guide.rst

```diff
@@ -65,22 +52,22 @@ RelativeHybridTopologyProtocol usage notes
 For production use of this protocol, we recommend the default settings, with these changes to reduce execution times per :external+gufe:py:class:`~gufe.transformations.Transformation` :py:class:`~alchemiscale.storage.models.Task`::
 
     >>> from openfe.protocols.openmm_rfe import RelativeHybridTopologyProtocol
+    >>> from openff.units import unit
 
     >>> settings = RelativeHybridTopologyProtocol.default_settings()
-    >>> settings.simulation_settings.equilibration_length = 1000 * unit.picosecond
-    >>> settings.simulation_settings.production_length = 5000 * unit.picosecond
-    >>> settings.alchemical_sampler_settings.n_repeats = 1
-    >>> settings.simulation_settings.output_indices = "not water"
-    >>> settings.engine_settings.compute_platform = "CUDA"
```

Comment:
also keep this? otherwise you'll get really bad performance issues if you accidentally have a broken GPU runner